packages/omg-js-childchain/src/transaction/utxoToInput.js

```diff
@@ -11,4 +11,20 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License. */
\ No newline at end of file
+limitations under the License. */
+
+const { InvalidArgumentError } = require('@omisego/omg-js-util')
+
+function utxoToInput (fromUtxo, toAddress, toAmount) {
```

Comment:
I think the name `utxoToInput` is not really accurate, as it's actually creating a transactionBody from a utxo, an address and an amount. How about `createTransactionBody`?
Also, this function could probably go into the `transaction.js` file

Reply:
yup, that sounds more consistent